docs/source/sky-computing.rst

```diff
@@ -29,14 +29,14 @@ regions/clusters with the lowest cost" or "monitor and manage jobs on both AWS a
 - Valuable engineering hours are invested to mask the differences across infra choices.
 - Workloads are forced to run on suboptimal choices (to heuristically simplify the search space), wasting utilization, cost savings, and capacity.
 
-Sky Computing
+Sky computing
```

Comment:
I think we can leave this one too. Zongheng made this change so I think the capitalization is intentional.
We should be able to just avoid all the changes to this file.